doc/kubernetes/modules/ROOT/pages/running/switch-back.adoc

```diff
@@ -0,0 +1,115 @@
+
+{description}
+
+// Infinispan -> trigger state transfer secondary -> primary (assumes empty caches)
+// Aurora -> switch to primary site?
+// Keycloak -> switch to primary site?
+// Route53 -> switch to primary site
+
+== Audience
+
+This guide contains the procedures required to bring the primary site back to operation when the secondary site is handling all the traffic.
+At the end of the guide, the primary site is online again and handles the traffic.
+
+See xref:running/index.adoc[] for additional guides.
+
+== Prerequisites
+
+This assumes a xref:running/deployments/active-passive-sync.adoc[] setup.
+
+== Procedures
+
+=== {ispn} Cluster
+
+For the context of this guide, `{site-a}` is the primary site, recovering back to operation, and `{site-b}` is the secondary site, running in production.
+
+include::partial$running/infinispan-batchcr-intro.adoc[]
+
+The `ConfigMap` with the {ispn} CLI operations are as follows:
+
+.Checks the cross-site status for all caches backing up to `{site-a}`.
+[source,yaml]
+----
+include::example$helm/ispn-site-b.yaml[tag=infinispan-crossdc-status]
+----
+
+.Starts a cross-site state transfer for all caches backing up to `{site-a}`.
+[source,yaml]
+----
+include::example$helm/ispn-site-b.yaml[tag=infinispan-crossdc-push-state]
+----
+
+.Checks the cross-site state transfer progress.
+[source,yaml]
+----
+include::example$helm/ispn-site-b.yaml[tag=infinispan-crossdc-push-state-status]
+----
+
+.Resets the cross-site state transfer progress after its completion.
+[source,yaml]
+----
+include::example$helm/ispn-site-b.yaml[tag=infinispan-crossdc-reset-push-state-status]
+----
+
+.Clears the cache data in `{site-a}`.
+[source,yaml]
+----
+include::example$helm/ispn-site-a.yaml[tag=infinispan-crossdc-clear-caches]
+----
+
+[#running-infinispan-batch]
+==== Running {ispn} Batch
+
+include::partial$running/infinispan-running-batch.adoc[]
+
+==== Procedures
+
+After the {ispn} in the primary site is back online and joined the cross-site channel (see xref:running/infinispan-crossdc-deployment.adoc#verifying-the-deployment[verifying the {ispn} deployment]), the state transfer must be manually started from the secondary site.
+
+After clearing the state in the primary site, it transfers the full state from the secondary site to the primary site, and it must be completed before the primary site can start handling incoming requests.
+
+WARNING: Transferring the full state may impact the {ispn} cluster perform by increasing the response time and/or resources usage.
+
+The first procedure is to delete any stale data from the primary site.
+
+. Login to the primary site.
+
+. Shutdown Keycloak. This will clear all Keycloak caches, and it prevents the state of Keycloak from being out-of-sync with Infinispan.
+
+. Run the `crossdc-clear-caches` Batch CR (see xref:#running-infinispan-batch[]).
+
+. Delete all Batch CRs created.
+
+Now we are ready to transfer the state from the secondary site to the secondary site.
```

Comment:
"from the secondary site to the primary site".

Reply:
Thanks, fixed in next commit!